trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationPlanetaryService.java

```diff
@@ -16,8 +13,9 @@
 @Slf4j
 public class RequestStationPlanetaryService implements RequestDataService<Station> {
 
-    private final RequestDataMessageRepository requestDataMessageRepository;
-    private final ObjectMapper objectMapper;
+    public static final String TOPIC = "stationPlanetaryDataRequest";
+    public static final String REQUESTING_MODULE = "trade";
```

Comment:
The "requesting module" is actually something I was wondering when working on POJOS to records conversion. It's hardly a very dynamic piece of information and certainly walks and quacks like an `enum` to me. 🤔